src/software/simulation/simulator_robot_singleton_test.cpp

```diff
@@ -21,6 +24,11 @@ extern "C"
 class SimulatorRobotSingletonTest : public testing::Test
 {
    protected:
+    void SetUp() override
+    {
+        app_logger_init(0, &TestUtil::handleTestRobotLog);
```

Comment:
`SimulatorRobotSingleton` has a `handleYellowRobotLogProto`, why can't we use that?

Reply:
A bit more documentation would help

Reply:
added comment here, and created a ticket to remove the duplication